ledger_device_sdk/src/ui/gadgets.rs

```diff
@@ -568,7 +578,17 @@ impl<'a> MultiPageMenu<'a> {
                     }
                 },
                 io::Event::Command(ins) => return EventOrPageIndex::Event(io::Event::Command(ins)),
-                _ => (),
+                io::Event::Ticker => {
```

Comment:
Can we have something in the `io` module that would work also for NBGL based apps ?